umbra-js/src/classes/Umbra.ts

```diff
@@ -56,6 +57,7 @@ const chainConfigs: Record<number, ChainConfig> = {
   100: { chainId: 100, umbraAddress, batchSendAddress, startBlock: 28237950, subgraphUrl: subgraphs[100] }, // Gnosis Chain
   137: { chainId: 137, umbraAddress, batchSendAddress, startBlock: 20717318, subgraphUrl: subgraphs[137] }, // Polygon
   1337: { chainId: 1337, umbraAddress, batchSendAddress, startBlock: 8505089, subgraphUrl: false }, // Local
+  8453: { chainId: 8453, umbraAddress, batchSendAddress, startBlock: 0, subgraphUrl: subgraphs[8453] }, // Base
```

Comment:
Should start block be when Umbra was deployed on Base? @garyghayrat

Reply:
yes!

Reply:
Thanks guys!  Done: https://github.com/ScopeLift/umbra-protocol/pull/643/commits/a368b4a7ab463e43ae5e4feb7b7d86a592500f96